### ISSUE TITLE
LineLength cannot be disabled in javascript: blocks

### ISSUE THREAD

**Issue by gregnavis** [sds/haml-lint#623]
I'm using Ruby 3.3.0, `haml` 7.0.2, and `haml-lint` 0.68.0.

### Steps to Reproduce

Lint the following file with default `haml-lint` settings:

```haml
-# haml-lint:disable LineLength
:javascript
  let thisIsLongButDescriptiveVariableNameWhoseSolePurposeIsToDemonstrateTheIssueBeingReportedAlthoughNoOneShouldNameVariablesLikeThatInPractice = true;
```

### Expected Result

Line length errors are not reported for the entire content of the JavaScript block.

### Actual Result

An error is being reported:

```
app/views/account/show.html.haml:5 [W] LineLength: Line is too long. [152/80]
```
